app/assets/stylesheets/bootstrap-overrides.scss

```diff
@@ -0,0 +1,59 @@
+// Bootstrap overrides that require bootstrap defined mixins
+
+// Buttons
+.btn-primary {
+  @include button-variant(
+    $sul-btn-primary-color,
+    darken($sul-btn-primary-color, 7.5%),
+    darken($sul-btn-primary-color, 10%),
+    lighten($sul-btn-primary-color, 5%),
+    lighten($sul-btn-primary-color, 10%),
+    darken($sul-btn-primary-color, 30%)
+  );
+}
```

Comment:
I'm surprised that we have to do this, but overriding the various `btn-*` variables doesn't appear to work in BS4. Happy to explore other alternatives if anybody is aware of them.

Reply:
I think you have to either do this, or override the colors in the variables (if I remember correctly).

https://github.com/sul-dlss/earthworks/blob/master/app/assets/stylesheets/bootstrap-variables.scss#L4